### PR TITLE
[TDV-147] Add reset account role to unverified action

### PR DESCRIPTION
**Pull request by ardier16**
+ reset to unverified action
+ minor fixes

### CODE REVIEW

File: src/components/User/Users/components/UserDetails/UserDetails.Request.vue [hunk 1]
```diff
@@ -1,29 +1,43 @@
...
+            @click="showRejectModal"
+            :disabled="isPending"
+          >
+            Reject
+          </button>
+        </div>
+      </template>
+    </template>
+
+    <template v-else-if="canRoleBeReset">
```
Comment (max-nes) [inline src/components/User/Users/components/UserDetails/UserDetails.Request.vue]: `canResetRole`

File: src/components/User/Users/components/UserDetails/UserDetails.Request.vue [hunk 2]
```diff
@@ -1,29 +1,43 @@
...
+      <button
+        class="app__btn
+               app__btn--danger
+               user-request__reset-to-unverified-btn"
+        @click="showRejectModal"
```
Comment (max-nes) [inline src/components/User/Users/components/UserDetails/UserDetails.Request.vue]: make a separate modal for resetting. their UI is similar, but the business logic is completely different, we shouldn’t mess them

also your button showing logic gonna be easier

File: src/components/User/Users/components/UserDetails/UserDetails.Request.vue [hunk 3]
```diff
@@ -154,13 +181,56 @@ export default {
...
+      }
+      this.isPending = true
+      try {
+        const operation = Sdk.base.CreateChangeRoleRequestBuilder
+          .createChangeRoleRequest({
+            requestID: '0',
+            destinationAccount: this.user.address,
+            accountRoleToSet: config.ACCOUNT_ROLES.notVerified.toString(),
+            creatorDetails: {
+              reason: this.rejectForm.reason,
```
Comment (max-nes) [inline src/components/User/Users/components/UserDetails/UserDetails.Request.vue]: `resetToUnverifiedReason`

File: src/components/User/Users/components/UserDetails/UserDetails.Request.vue [hunk 4]
```diff
@@ -154,13 +181,56 @@ export default {
...
     showRejectModal () {
       this.rejectForm.reason = '' + EMPTY_REASON
       this.rejectForm.isShown = true
     },
 
     hideRejectModal () {
       this.rejectForm.isShown = false
+    },
+
+    async submitRejectForm () {
```
Comment (max-nes) [inline src/components/User/Users/components/UserDetails/UserDetails.Request.vue]: separate

File: src/components/User/Users/components/UserDetails/UserDetails.Request.vue [hunk 5]
```diff
@@ -202,4 +272,9 @@ export default {
 .user-details-request__form-actions {
   max-width: 48rem;
 }
+
+.user-request__reset-to-unverified-btn {
+  max-width: 23.5rem;
```
Comment (max-nes) [inline src/components/User/Users/components/UserDetails/UserDetails.Request.vue]: is not it _too_ wide? maybe something like 15rem?

File: src/components/User/Users/components/UserDetails/UserDetails.Request.vue (hunk 1 above)
Comment (ardier16) [reply src/components/User/Users/components/UserDetails/UserDetails.Request.vue]: Renamed

File: src/components/User/Users/components/UserDetails/UserDetails.Request.vue (hunk 2 above)
Comment (ardier16) [reply src/components/User/Users/components/UserDetails/UserDetails.Request.vue]: Extracted

File: src/components/User/Users/components/UserDetails/UserDetails.Request.vue (hunk 4 above)
Comment (ardier16) [reply src/components/User/Users/components/UserDetails/UserDetails.Request.vue]: Extracted

File: src/components/User/Users/components/UserDetails/UserDetails.Request.vue (hunk 5 above)
Comment (ardier16) [reply src/components/User/Users/components/UserDetails/UserDetails.Request.vue]: Updated

**Pull request merged**